holoviews/streams.py

```diff
@@ -14,6 +14,9 @@
 
 from contextlib import contextmanager
 
+# Types supported by Pointer derived streams
+pointer_types = (Number, util.basestring, tuple)+util.datetime_types
```

Comment:
Super minor comment: spaces around operators like ``+`` are preferred unless you are clearly trying to save horizontal space.